css/main.css

```diff
@@ -1,3 +1,203 @@
-/* Add your custom CSS here */
 
-/* Dont' forget to link this file to your HTML in the <head> */
+.hamIcon{
+  position: absolute;
+  right: 10px;
+  top: 20px;
+}
+
+.header_customize{
+  background-color:#466381;
+  justify-content:space-around;
+  font-family: 'Vast Shadow', cursive;
+  display:flex;
+}
+.logo_img{
+  width:10%;
+}
+.ul_header {
+  list-style-type: none;
+  margin-top: 30px;
+}
+.ul_header_nav{
+  display: flex;
+  list-style-type: none;
+  color: black;
+  margin-top: 50px;
+}
+ul.ul_header_nav a {
+  padding-right: 10px;
+  padding-left: 20px;
+  margin-top: 40px;
+  color: black;
+  text-decoration: none;
+}
+
+ul.ul_header_nav a:hover{
+  color: red;
+}
+
+.logo_img{
+  background: url("steve-jobs-nav.jpg");
+
+}
+
+.hero_image{
+  background: url("../s.jpg");
+  background-size:cover;
+  background-position:center;
+  background-repeat: no-repeat;
+  min-height:120vh;
+}
+
+.font_anton{
+  font-family: 'Anton', sans-serif;
+  text-align: center;
+  color: blanchedalmond;
+  font-size: 100px;
+  margin: 0;
+
+}
+
+.section_one{
+  display: flex;
+  flex-direction: column;
+  justify-content:flex-start;
+
+}
+
+
+.font_section_one{
+  display: flex;
+  align-items: flex-start;
+  flex-direction: row;
+  justify-content: center; 
+}
+
+.button{
+  border: none;
+  color: white;
+  text-align: center;
+  text-decoration: none;
+  font-size: 16px;
+  cursor: pointer;
+  background-color: #466381;
+  margin: 0 auto;
+  padding: 10px 10px;
+}
+ .p_section2{
+   display: flex;
+   flex-direction:column;
+ }
+
+.section_two{
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  font-family: 'Rokkitt', serif;
+}
+.about_me{
+  width:100%;
+  background:url("apple_sign.jpg") ;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding:10px 10px;
+  text-align: center;
+  font-size: 20px;
+}
+
+.button_about_me{
+  border: none;
+  color: white;
+  padding: 15px 20px;
+  text-align: center;
+  text-decoration: none;
+  background-color: #466381;
+  cursor: pointer;
+  margin-top: 5px;
+}
+.about_me_img{
+  display: flex;
+  justify-content: flex-start;
+  text-align:flex-start;
+  background-repeat: no-repeat;
+  background-size: cover;
```

Comment:
These rules won't work because your image is not a background-image. 
The image for this is actually inside an `img` tag in your HTML, and it's getting distorted at the moment. Remove that and set the image to be a background-image, like you did for the Steve Jobs picture. That way, it will cover the whole space and scale much better.